contracts/crowdsale/emission/MintedCrowdsale.sol

```diff
@@ -22,6 +22,7 @@ contract MintedCrowdsale is Crowdsale {
   )
     internal
   {
-    require(MintableToken(token).mint(_beneficiary, _tokenAmount));
+    // Potentially dangerous assumption about the type of the token.
+    require(MintableToken(address(token)).mint(_beneficiary, _tokenAmount));
```

Comment:
Why is the cast to `address` necessary? Can't an `ERC20` token be casted to `MintableToken`?

Reply:
See the linked issue above. "Incompatible" contracts won't be possible to be typecasted, but going through the `address` type lifts those safeguards.

Reply:
The safe way to do this would be to define a constructor in `MintedCrowdsale` which requires `MintableToken` as an input for `token`. You'd still need the above change, but the contract would be safe 😉

Reply:
When you say 'safe', are you talking about `token` not actually being a `MintableToken`? Is there any security-related change in 0.5.0 regarding these types of casts (besides what you just mentioned)?